crml/governance/src/lib.rs

```diff
@@ -43,9 +43,9 @@ use sp_std::prelude::*;
 /// Identifies governance scheduled calls
 const GOVERNANCE_ID: LockIdentifier = *b"governan";
 /// The length in blocks of a referendum voting cycle
-const REFERENDUM_LENGTH: u32 = 10;//2000;
+const REFERENDUM_LENGTH: u32 = 2000;
```

Comment:
`7200` blocks every 24 hours on average
3 day default makes sense to me: `3 * 7200`

try refactor this const like this:
https://github.com/cennznet/cennznet/blob/928509f4b8fc2002fd3474a97465902f4a32ed97/crml/nft/src/lib.rs#L68
allows it to be set lower for testing in mock runtime

Reply:
can do this later when setting up tests